2022-02-04 03:30:11,684 INFO    MainThread:17372 [wandb_setup.py:_flush():71] setting env: {}
2022-02-04 03:30:11,685 INFO    MainThread:17372 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-04 03:30:11,685 INFO    MainThread:17372 [wandb_init.py:_log_setup():371] Logging user logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_033011-3n1ituj0\logs\debug.log
2022-02-04 03:30:11,686 INFO    MainThread:17372 [wandb_init.py:_log_setup():372] Logging internal logs to D:\PythonProject\Time_series_anomaly_detection\wandb\run-20220204_033011-3n1ituj0\logs\debug-internal.log
2022-02-04 03:30:11,686 INFO    MainThread:17372 [wandb_init.py:init():404] calling init triggers
2022-02-04 03:30:11,686 INFO    MainThread:17372 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-04 03:30:11,686 INFO    MainThread:17372 [wandb_init.py:init():460] starting backend
2022-02-04 03:30:11,687 INFO    MainThread:17372 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=spawn, using: spawn
2022-02-04 03:30:11,697 INFO    MainThread:17372 [backend.py:ensure_launched():216] starting backend process...
2022-02-04 03:30:11,819 INFO    MainThread:17372 [backend.py:ensure_launched():222] started backend process with pid: 19344
2022-02-04 03:30:11,820 INFO    MainThread:17372 [wandb_init.py:init():469] backend started and connected
2022-02-04 03:30:11,822 INFO    MainThread:17372 [wandb_init.py:init():533] updated telemetry
2022-02-04 03:30:11,929 INFO    MainThread:17372 [wandb_init.py:init():563] communicating current version
2022-02-04 03:30:13,740 INFO    MainThread:17372 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-04 03:30:13,742 INFO    MainThread:17372 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-04 03:30:14,138 INFO    MainThread:17372 [wandb_init.py:init():606] starting run threads in backend
2022-02-04 03:30:17,169 INFO    MainThread:17372 [wandb_run.py:_console_start():1810] atexit reg
2022-02-04 03:30:17,182 INFO    MainThread:17372 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-04 03:30:17,182 INFO    MainThread:17372 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-04 03:30:17,184 INFO    MainThread:17372 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-04 03:30:17,184 INFO    MainThread:17372 [wandb_init.py:init():633] run started, returning control to user process
2022-02-04 03:30:17,831 INFO    MainThread:17372 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-04 03:30:17,832 INFO    MainThread:17372 [wandb_run.py:_restore():1752] restore
2022-02-04 03:30:18,773 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 03:30:19,197 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 710
}

2022-02-04 03:30:19,302 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:30:19,411 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:30:19,520 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:30:19,639 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:30:19,743 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 710
  total_bytes: 6760
}

2022-02-04 03:30:19,852 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3914
  total_bytes: 6760
}

2022-02-04 03:30:19,964 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 3914
  total_bytes: 6760
}

2022-02-04 03:30:20,074 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:30:20,184 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:30:20,296 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:30:20,407 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:30:20,516 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:30:21,106 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}

2022-02-04 03:30:22,025 INFO    MainThread:17372 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 6760
  total_bytes: 6760
}
local_info {
}

2022-02-04 03:30:23,227 INFO    MainThread:17372 [wandb_run.py:_append_files():2180] logging synced files
